indragunawan/api-rate-limit-bundle/0.2/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
+        "Indragunawan\\ApiRateLimitBundle\\IndragunawanApiRateLimitBundle": ["all"]
```

Comment:
What if we only activate the bundle in "prod"?

Reply:
done